src/recipe.yaml

```diff
@@ -1141,3 +1141,19 @@
     model_name: meta-llama/Llama-3.3-70B-Instruct
   extra_envs:
     - name: HF_TOKEN
+"deepseek:v3":
+  project: vllm-chat
+  service_config:
+    name: deepseek-v3
+    traffic:
+      timeout: 300
+    resources:
+      gpu: 16
+      gpu_type: nvidia-a100-80gb
```

Comment:
Hmm I think we should support H100 here.

cc @bojiang for supporting upstream with bentoml?